From 13ab4b7fa6f92eb9819a01129c4e4a0a9c401ee8 Mon Sep 17 00:00:00 2001
From: Xie XiuQi <xiexiuqi@huawei.com>
Date: Wed, 7 Jul 2021 16:22:56 +0800
Subject: [PATCH] arm64: errata: enable HISILICON_ERRATUM_HIP08_RU_PREFETCH
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 13ab4b7fa6f92eb9819a01129c4e4a0a9c401ee8


hulk inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I3ZFV2
CVE: NA

------------------------------------------------------------

enable CONFIG_HISILICON_ERRATUM_HIP08_RU_PREFETCH, to add a
cmdline option to disable prefetch.

Signed-off-by: Xie XiuQi <xiexiuqi@huawei.com>
Reviewed-by: Hanjun Guo <guohanjun@huawei.com>
Signed-off-by: Cheng Jian <cj.chengjian@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/arm64/configs/openeuler_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index 1b442b1d129b..b60bed463a8e 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -358,6 +358,8 @@ CONFIG_QCOM_FALKOR_ERRATUM_1003=y
 CONFIG_QCOM_FALKOR_ERRATUM_1009=y
 CONFIG_QCOM_QDF2400_ERRATUM_0065=y
 CONFIG_QCOM_FALKOR_ERRATUM_E1041=y
+CONFIG_HISILICON_ERRATUM_HIP08_RU_PREFETCH=y
+# CONFIG_HISILICON_HIP08_RU_PREFETCH_DEFAULT_OFF is not set
 CONFIG_SOCIONEXT_SYNQUACER_PREITS=y
 # end of ARM errata workarounds via the alternatives framework
 
-- 
2.26.2

